src/DrDotnet.Profilers/profilers/cpu_hotpath_profiler.rs

```diff
@@ -33,15 +35,32 @@ impl CpuHotpathProfiler {
     fn print_callstack(profiler_info: ClrProfilerInfo, calls: std::sync::MutexGuard<DashMap<usize, AtomicIsize>>)
     {
         let pinfo = profiler_info.clone();
+
+        // Variables to debug do_stack_snapshot calls
+        let mut nbThreads = 0;
+        let mut errors: HashMap<HRESULT, i32> = HashMap::new();
```

Comment:
The hashmap is going to introduce a lot of overhead just for the sake of debugging. 
It would be nice to use `#[cfg(debug_assertions)]` for instance (or maybe a custom macro to have blocks of code run debug configuration only)

Reply:
all good for testing purpose